File: src/arrays_and_strings/java/URLify.java
package arrays_and_strings.java;

import java.util.Scanner;

/**
 *
 */
public class URLify {
  public static void main(String[] args) {
    Scanner in = new Scanner(System.in);
    String str = in.nextLine();
    int len = in.nextInt();
    System.out.println(urlify(str, len));
  }

  private static String urlify(String str, int length) {
    return str.substring(0, length).replace(" ", "%20");
  }
}

File: src/arrays_and_strings/kotlin/URLify.kt
package arrays_and_strings.kotlin

/**
 *
 */
fun main(args: Array<String>) {
  val str = readLine().toString()
  val len = readLine().toString().toInt()

  print(urlify(str, len))
}

fun urlify(str: String, len: Int): String {
  return str.substring(0, len).replace(" ", "%20")
}